### PR TITLE
Add quit step to Manual Deployment docs

### LINKED ISSUE #614: Explain how to use `quit` with a secure cluster
An example with `quit` is missing from https://www.cockroachlabs.com/docs/manual-deployment.html#deploy-a-secure-cluster.

Since `quit` needs to connect to the node it probably needs a certificate too. An example proper command line to do so would be welcome.

Reported by @HenryEscobar

### PR DESCRIPTION
**Pull request by jseldess**
Add step for stopping nodes.

Fixes #614

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/docs/619)

### CODE REVIEW

File: manual-deployment.md [hunk 1]
```diff
@@ -221,6 +233,18 @@ store[0]:  path=cockroach-data
 
 {{site.data.alerts.callout_info}}In cases where you set <code>--http-addr</code> to <code>localhost</code> and need to access the Admin UI from a separate machine, you can use SSH to tunnel from the machine to a node.{{site.data.alerts.end}}  
 
+### 9. Stop the cluster
+
+You can stop the nodes (and therefore the cluster) from any machine with the `cockroach` binary, the CA cert, and a client certificate and key. Make sure the machine you want to use has these files and then stop the nodes:  
+
+~~~ shell
+$ cockroach quit --host=<node1-hostname> --ca-cert=ca.cert --cert=root.cert --key=root.key
```
Comment (mberhault) [inline manual-deployment.md]: I assume you checked that calling `quit` with user `root` works?

**Comment by sploiselle** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-Vn_cfEdM2nDTmT8V:b-ytimse) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
>  --key=root.key
should this be `--key=node1.key`? and the following two `node2.key` and `node3.key` respectively?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by mberhault** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-XnUnmUNhbpZVUt-A:b-gmq2xs) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary>
> >  --key=root.key
> should this be `--key=node1.key`? and the following two `node2.key` and `node3.key` respectively?
</details>
it would be those keys if we were using the node certificate, but we definitely want it to work as the `root` user

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by sploiselle** [comment]
Review status: 0 of 1 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-fmwHknDa7khbZuXB:b-fay2cd) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary>
> it would be those keys if we were using the node certificate, but we definitely want it to work as the `root` user
</details>
But the second and third nodes are quit using `node1.key` even though they wouldn't have that file stored on them? Or should all of them use `root.key`?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by jseldess** [comment]
*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-jbV4XwdxD5biizkb:b-1sf4a3) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, sploiselle (Sean Loiselle) wrote…</i></summary>
> But the second and third nodes are quit using `node1.key` even though they wouldn't have that file stored on them? Or should all of them use `root.key`?
</details>
That's a mistake, @sploiselle. I'll fix that now.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by jseldess** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-nfYNwHkwhO8cHvnA:b-3i09j2) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, jseldess wrote…</i></summary>
> That's a mistake, @sploiselle. I'll fix that now.
</details>
@mberhault, `quit` works with the cert and key for the `root` user on a local secure cluster. But I haven't tested this on GCE. Would it be any different?

@sploiselle, let me know if/how you'd suggest clarifying this step. For example, should add alternate commands if you are quitting from one of the node machines? 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by mberhault** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-u_Zq1FlLOkH11Plp:b-dvzo70) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, jseldess wrote…</i></summary>
> @mberhault, `quit` works with the cert and key for the `root` user on a local secure cluster. But I haven't tested this on GCE. Would it be any different?
> 
> @sploiselle, let me know if/how you'd suggest clarifying this step. For example, should add alternate commands if you are quitting from one of the node machines? 
</details>
quitting the node on the local machine could be done with the node cert/key, but I'm not sure I would recommend it. It's easier to just send a signal to the process.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by mberhault** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR-uuaXuWHXDZGt2qIs:b-n68e1m) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary>
> quitting the node on the local machine could be done with the node cert/key, but I'm not sure I would recommend it. It's easier to just send a signal to the process.
</details>
it should work just fine on GCE. as long at the root cert/key can be used to talk to the cluster as an application, it should work just fine for `cockroach quit`

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Comment by jseldess** [comment]
*[manual-deployment.md, line 241 \[r1\]](https://reviewable.io:443/reviews/cockroachdb/docs/619#-KR-Uw_1ON1Tz76KePUM-r1-241:-KR44HU1nmJBouRNNPjQ:b4qk6r9) ([raw file](https://github.com/cockroachdb/docs/blob/db02629d76675d9f2d654f7c48a7db389850137b/manual-deployment.md#L241)):*
<details><summary><i>Previously, mberhault (marc) wrote…</i></summary>
> it should work just fine on GCE. as long at the root cert/key can be used to talk to the cluster as an application, it should work just fine for `cockroach quit`
</details>
OK, given that, am I good to merge now? 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/619)*

**Pull request merged**